osu.Game/Screens/Play/HUD/HealthDisplay.cs

```diff
@@ -52,14 +58,46 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
-            Current.BindTo(HealthProcessor.Health);
             HealthProcessor.NewJudgement += onNewJudgement;
 
+            // Don't bind directly so we can animate the startup procedure.
+            health = HealthProcessor.Health.GetBoundCopy();
+            health.BindValueChanged(h =>
+            {
+                Current.Value = h.NewValue;
+                finishInitialAnimation();
+            });
```

Comment:
The issue with putting this in `HealthDisplay` that there's an odd one out here and it's `FailingLayer` which responsible for the red glow on the sides of the playfield. Letting it share this logic leads to it doing this:

https://github.com/ppy/osu/assets/20418176/8ddfdcb9-8022-4b0f-8766-411d5ea888eb

which probably isn't good.

Reply:
Motherf------

I originally put this in `HealthProcessor` and moved it out thinking that the `FailingLayer` would *definitely* not be a health display.

I may end up just moving the to per-implementation. Unless you have any better ideas. In the future I can imagine individual implementations having different "startup" animations.